sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SyntaxTrackers/InvocationTracker.cs

```diff
@@ -116,6 +116,10 @@ bool IsTrackedMethod(SyntaxNode invocation, SemanticModel semanticModel)
                 context.MethodSymbol.Value != null &&
                 context.MethodSymbol.Value.ReturnType.DerivesFrom(returnType);
 
+        public InvocationCondition IsInvalidBuilderInitialization<TInvocationSyntax>(BuilderPatternCondition<TInvocationSyntax> condition)
+            where TInvocationSyntax : SyntaxNode =>
+            context => condition.InvalidBuilderInitialization(context);
```

Comment:
`context => condition.InvalidBuilderInitialization(context);` can be replaced with `condition.InvalidBuilderInitialization;`